benchtable.js

```diff
@@ -160,7 +161,12 @@ BenchTable.prototype = Object.create(Benchmark.Suite.prototype);
 BenchTable.prototype.run = function(config) {
   var self = this, mapkey, mapfun;
 
-  function createBenchmarkFunction(fun, input) {
+  function createBenchmarkFunction(fun, options, input) {
```

Comment:
What do you think about moving the `options` parameter to the last place? Just wondering if there's a reason you put it in 2nd place here, and in last place in `addFunction`. :thought_balloon: